Formula/librdkafka.rb

```diff
@@ -13,11 +13,12 @@ class Librdkafka < Formula
   end
 
   depends_on "lzlib"
+  depends_on "openssl"
 
   def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
-    system "make"
+    system "C_INCLUDE_PATH=.:#{include}:#{Formula["openssl"].opt_prefix}/include make"
```

Comment:
I found it necessary. I'm not sure what was the difference in our implementations.